src/components/TransactionItemRow/DataCells/ReceiptCell.tsx

```diff
@@ -25,8 +24,7 @@ function ReceiptCell({transactionItem, isSelected, style}: {transactionItem: Tra
     if (source && typeof source === 'string') {
         const filename = getFileName(source);
         const receiptURIs = getThumbnailAndImageURIs(transactionItem, null, filename);
-        const isReceiptPDF = Str.isPDF(filename);
-        source = tryResolveUrlFromApiRoot(isReceiptPDF && !receiptURIs.isLocalFile ? (receiptURIs.thumbnail ?? '') : (receiptURIs.image ?? ''));
```

Comment:
Don't we still need the `isLocalFile` check? Doesn't that mean the file is local and thus we don't have thumbnails since those are generated in the server?